components/rmrk/Create/CreateToken.vue

```diff
@@ -271,7 +268,7 @@ export default class CreateToken extends mixins(
             setTimeout(
               () =>
                 this.listForSale(
-                  createdNFTs.value,
+                  createdNFTs.value[0],
```

Comment:
This basically prevents listing of multiple NFTs

Reply:
but this page doesn't mint multiple nft's anyways
we (will) have massmint for that

Reply:
Edition cries in the corner

<img width="713" alt="Screenshot 2023-06-16 at 15 47 11" src="https://github.com/kodadot/nft-gallery/assets/22471030/dc6d4cf1-676c-4b1d-aa7f-2755e5236921">